pages/workspace/connections.mdx

```diff
@@ -21,6 +21,20 @@ Avo currently supports these development platforms and programming languages:
 ### <a name="web"></a> Web
 
 - JavaScript
+  - Segment
+
+Amplitude
+
+Custom
+
+Mixpanel
+
+mParticle
+
+Snowplow
+
+Facebook Analytics
+
```

Comment:
Something weird happening with the formatting here 👀

Reply:
Ah, here I thought to list the destinations for each source, but then decided to not do it because it turned out to be too complex and potentially hard to maintain.